src/CookiePlugin.php

```diff
@@ -86,6 +87,7 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
      * @param $setCookie
      *
      * @return Cookie|null
```

Comment:
the styleci complaint is here: we require a blank line between the `@return` and an `@throws`